extensions/wikidata/tests/src/org/openrefine/wikidata/updates/ItemUpdateTest.java

```diff
@@ -78,36 +78,36 @@ public ItemUpdateTest() {
 
```

Comment:
This test class (`ItemUpdateTest`) should also be renamed to `TermedStatementEntityUpdateTest`.